.github/workflows/build.yml

```diff
@@ -0,0 +1,81 @@
+name: build
+on:
+  push:
+#    branches: [ master ] Todo: Uncomment before merging
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release? yes/no'
+        default: 'no'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Why do we want/need this? 

Why the question: In `jira-actions` we are using it to allow other parallel jobs to finish, so that we have results from other CI-level parametrised tests. Here I don't see us using `strategy.matrix`.

BTW I personally think we shouldn't setup parallelism on the CI level. It leads to differentiation of CI from local devloop which has it's negative consequences. Possibly we'd be able to shift it in `jira-actions` back to Gradle/junit level.

Reply:
Oh, this change was not intentional. I'll remove it.